### PR TITLE
Terms and definition

### LINKED ISSUE #105: Proper Sources
We have, unfortunately, many statements in our SRS that are clearly taken from somewhere without the explicit statement of a source.

As we are trying to contribute to science, this is not acceptable. Please state your sources for as many statements as possible.

Reading through Palldio's [literature list](http://www.palladio-simulator.com/science/literature_and_documentation/) will give you a good starting point to find scientific paper about Palladio and the component-base software development process. Using and citing them will not only convey a more professional image, it will also assure that we are using definitions compatible to Palladio and scientifically proven.

### PR DESCRIPTION
**Pull request by RomanLangrehr**
Results from my try to fix #105.

(All sources I found in the remaing time were not usefull for this)

### CODE REVIEW

File: doc/Requirements Specification/Parts/Terms and Definitions.lyx [hunk 1]
```diff
@@ -1262,6 +1264,7 @@ gls{PCM}, software architects create the assembly model, specifying how
  existing 
 \backslash
 glspl{component} are composed.
+
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: This line adds a missing space ;)

**Comment by acBerger** [comment]
lgtm

File: doc/Requirements Specification/Parts/Terms and Definitions.lyx [hunk 2]
```diff
@@ -735,10 +735,12 @@ newglossaryentry{Palladio}{
...
 
-		an approach for the definition of 
+		an approach for the prediction of 
 \backslash
-glspl{component-based software architecture} with a special focus on performance
- properties.
+glspl{component-based software architecture} 
+\backslash
+gls{QoS} with a special focus on performance properties.
+
```
Comment (flyfl) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: >an approach for the prediction of component-based software

Shouldn't it be "an approach to predict performance properties of component-based software architecture"?

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: Yeah, that’s better!

**Comment by jGleitz** [comment]
lgtm if the above is fixed. If the next commit only fixes the above sentence, this may be merged without waiting for Travis.

**Pull request merged**